src/general-components/Session/Session.tsx

```diff
@@ -71,6 +71,21 @@ class Session {
     static removeTokens = () => {
         Session.token.delete();
         Session.refreshToken.delete();
+
+        // Darkmode
+        localStorage.removeItem("darkmode");
+
+        let settings = SettingsList.FromArray([{
+            name: "Dark Mode",
+            value: "false",
+            id: 1,
+            type: "",
+            extras: "",
+            description: "",
+            exists: true,
+            default: "false"
+        }]);
+        DarkModeChanger(settings, settings);
```

Comment:
Please move this into DarkModeChanger file and use a listener system to call this code.